packages/dpkg/build.sh

```diff
@@ -72,13 +71,21 @@ share/polkit-1
 "
 
 termux_step_pre_configure() {
+	# `dpkg`'s configure script expects the `.dist-version` file to contain the package version.
+	# This is missing in the tarballs from salsa.debian.org/dpkg-team/dpkg
+	echo "$TERMUX_PKG_VERSION" > "$TERMUX_PKG_SRCDIR/.dist-version"
+
+	# `.dist-vcs-id` isn't strictly required, but configure complains if its missing.
+	git ls-remote https://salsa.debian.org/dpkg-team/dpkg.git \
+	| grep -F "refs/tags/$TERMUX_PKG_VERSION^{}" \
+	| cut -f1 > "$TERMUX_PKG_SRCDIR/.dist-vcs-id"
 	(
 		cd "${TERMUX_PKG_SRCDIR}" && \
 		./autogen && \
 		patch -p1 -i "${TERMUX_PKG_BUILDER_DIR}"/configure.diff
```

Comment:
Probably we want to apply patch with expanded `TERMUX_ARCH` value, in this case we will not need to use `sed`/`perl` later in the script.
```suggestion
		sed -e "s/TERMUX_ARCH/${TERMUX_ARCH}/" "${TERMUX_PKG_BUILDER_DIR}/configure.diff" | patch --silent -p1
```

Reply:
No to this, since the suggestion above already does this, and I don't wanna touch this part of the build right now.

Reply:
Doing that right now.
I changed it to `@TERMUX_ARCH@` to follow the pattern we use for env var replacements in patches.